services/batch.rst

```diff
@@ -0,0 +1,582 @@
+
+To report system issues, log into `LSST JIRA <https://jira.lsstcorp.org/>`_ and file a `JIRA ticket in the IT Helpdesk Support <https://ls.st/ihsticket>`_ project tagging NCSA as the responsible organization.
+
+
+.. _batch-connect:
+
+LSST Batch Systems: Connecting and Authenticating
+=================================================
+
+The batch submit nodes can be accessed after first connecting to the :doc:`lsst-login <lsst-login>` nodes. Once connected to an ``lsst-login`` node a user can connect to a submit node via its short hostname (e.g., ``lsst-condordac-sub01``) without having to enter a password (Kerberos authentication should be used by default; if your Kerberos ticket expires on the login node you may need to ``kinit`` again before proceeding to the submit node).
+
+For various suggestions on streamlining connections through the ``lsst-login`` nodes ("jump host" configuration, port forwarding, Kerberos) see :doc:`related documentation <lsst-login>`.
+
+In order to use HTCondor commands on the submit nodes you must have a valid Kerberos ticket. The following commands may be helpful in working with Kerberos tickets on the submit nodes:
+
+.. code-block:: text
+
+    # list current Kerberos tickets
+    $ klist
+
+    # renew your current Kerberos ticket
+    $ kinit -R
+
+    # create a new Kerberos ticket
+    $ kinit
+
+If you using an ``lsst-login`` node as a "jump host" and are authenticating to a submit node using a Kerberos ticket from your local machine (workstation/laptop), you may not have a Kerberos ticket when you arrive on the submit node itself. You can ``kinit`` on the submit node as described. Alternatively may wish to configure ``GSSAPIDelegateCredentials yes`` in your local ``~/.ssh/config`` file in order to forward your Kerberos credentials to the submit node and automatically create a ticket there upon connection.
+
+
+.. _batch-gpfs:
+
+LSST Batch Systems: GPFS Directory Spaces
+=========================================
+
+The nodes in the batch clusters utilize the General Parallel File System (GPFS) to provide shared storage across all of the nodes.
+
+For convenience the bind mounts  :file:`/scratch` , :file:`/project` , :file:`/datasets` ,  and :file:`/software`  have been created to provide views into corresponding spaces in GPFS.
+
+Please see :doc:`Storage Resources <storage>` for more general information.
+
+To add/change/delete datasets, see :doc:`Common Dataset Organization and Policy </services/datasets>`.
+
+
+.. _batch-stack:
+
+LSST Batch Systems: Shared Software Stack in GPFS
+=================================================
+A shared software stack on the GPFS file systems, suitable for computation on the
+``Verification Cluster``, has been provided and is maintained by Science Pipelines and
+is available under :file:`/software/lsstsw`.  This stack may be initialized via:  ::
+
+     % .  /software/lsstsw/stack/loadLSST.bash
```

Comment:
I suggest using [this syntax](https://developer.lsst.io/restructuredtext/style.html#command-line-prompts) to make your example prompts match those elsewhere in the documentation.

If you _don't_ use that syntax, please change the line above to be:

> ... initialized via::

Which should have the same effect but be a bit less verbose than what you have now!